jenkins_pipelines/environments/manager-4.2-qe-sle-update

```diff
@@ -1,18 +1,7 @@
 #!/usr/bin/env groovy
 
 node('sumaform-cucumber-provo') {
-    def minionList = 'sle12sp4_client, sle12sp4_minion, sle12sp4_ssh_minion, ' +
```

Comment:
do we want to remove almost all of them for some specific reason? :grimacing:

Reply:
yes, the reason is that SLE updates test suite only has one minion

here too, I have been too lazy to open a separate PR